slices/php8.3-opcache.yaml

```
@@ -0,0 +1,18 @@
+package: php8.3-opcache
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - php-common_libs
+      - php8.3-common_libs
+      - php8.3-opcache_config
+      - ucf_bins
+    contents:
+      /usr/lib/php/20230831/opcache.so:
```

```suggestion
      /usr/lib/php/*/opcache.so:
```
I'm not sure how php versions modules in relation to php itself (ex. 8.3), but we could glob this just in case.